src/test/java/com/revaturelabs/ask/AskApplicationControllerTests.java

```diff
@@ -222,23 +312,28 @@ public void testUpdateTag() {
     assertEquals(exampleTag, tagControllerImpl.updateTag(exampleTag, 1));
   }
 
-
   /**
    * Test deleting tag.
    */
-  // @Test
-  // public void testDeleteTag() {
-  // Tag exampleTag = new Tag();
-  // exampleTag.setId(1);
-  // exampleTag.setName("JavaScript");
-  //
-  // when((tagServiceMock.create(exampleTag))).thenReturn(exampleTag);
-  // tagServiceMock.create(exampleTag);
-  // tagServiceMock.delete(1)
-  // when((tagServiceMock.getById(exampleTag.getId()))).thenReturn(null);
-  // assertEquals(null, tagControllerImpl.getTagById(1), null);
-  //
-  // }
+  @Test
+  public void testDeleteTag() {
+    Tag exampleTag = new Tag();
+    exampleTag.setId(1);
+    exampleTag.setName("JavaScript");
+
+    when((tagServiceMock.create(exampleTag))).thenReturn(exampleTag);
+    tagServiceMock.create(exampleTag);
+    doNothing().when(tagServiceMock).delete(exampleTag.getId());
+    tagServiceMock.delete(exampleTag.getId());
+    when((tagServiceMock.getById(exampleTag.getId()))).thenReturn(null);
+    tagControllerImpl.deleteTag(exampleTag.getId());
```

Comment:
@KRT0335 , this is an example of what I'm referencing. There are no assertions, and it's not evident what you're testing.

Reply:
`Assert.assertNull(tagServiceMock.getById(exampleTag.getId()));`
Was added in the following line.

Reply:
Mostly in the mindset that the tag no longer exists after deleting it.
It should be in the code after commit 3d3d7e18a9ec3a793e4fb7599b1876ba14958969
"made changes to tests, ignore one exception that can be tested, added…"